pkg/reconciler/labeler/v2/metasync.go

```diff
@@ -147,8 +147,8 @@ func clearMetaForNotListed(ctx context.Context, ns, routeName string, acc Access
 // element through the provided accessor.
 // A nil route name will cause the route to be de-referenced, and a non-nil route will cause
 // that route name to be attached to the element.
-func setRoutingMeta(ctx context.Context, acc Accessor, ns, name, routeName string, remove bool) error {
-	if mergePatch, err := acc.makeMetadataPatch(ns, name, routeName, remove); err != nil {
+func setRoutingMeta(ctx context.Context, acc Accessor, r *v1.Route, name string, remove bool) error {
+	if mergePatch, err := acc.makeMetadataPatch(r.Namespace, name, r.Name, remove); err != nil {
```

Comment:
well pass accessor to the makeMetadataPatch as well?

Reply:
Sure. Was trying to pass the route along more just to make it harder to accidentally swap "name" and "routeName" and simplify the param list